docs/modules/administration/pages/installation.adoc

```diff
@@ -52,7 +52,7 @@ make prepare-config
 
 If you want to install OpenDevStack for the first time, leave the BitBucket URL empty.
 
-After the configuration is prepared (which means that a directory `ods-configuration` will be created if it does not exist yet), fill out all the parameters for your installation and commit the result. You don't need to push yet, this will be done later.
+After the configuration is prepared (which means that a directory `ods-configuration` will be created if it does not exist yet), fill out all the parameters in `ods-core.env` in directory `ods-configuration` for your installation. Add all files of directory `ods-configuration` to git and commit the result. You don't need to push yet, this will be done later.
```

Comment:
"Preparing" also updates the sample files and checks if any params are present in the sample files but missing from the real file.

Reply:
will add it